sources/install/package_base.sh

```diff
@@ -396,6 +396,16 @@ function post_install() {
     if [ -d "/root/Desktop" ]; then rm -r /root/Desktop; fi
 }
 
+function uploader() {
+    colorecho "Installing Uploader"
+    git clone https://github.com/Frozenka/uploader.git /opt/tools/uploader
+    pip3 install -r /opt/tools/uploader/requirements.txt
```

Comment:
You need to use `venv`. For exemple you can add a `setup.py` / `pyproject.toml` file in the root of your repo (example: https://github.com/p0dalirius/smbclient-ng) and use `pipx` to install your tool.
Or, you can use a python venv -> https://exegol.readthedocs.io/en/latest/community/contributors.html